metafix/src/test/java/org/metafacture/metafix/MetafixRecordTest.java

```diff
@@ -2298,6 +2298,46 @@ public void transformRepeatedField() {
         );
     }
 
+    @Test
+    public void emitEntityForRepeatedField() {
+        MetafixTestHelpers.assertFix(streamReceiver, true, Arrays.asList(
```

Comment:
Is this such a common case that we need a whole new helper for it?

We can simply call `i.setRepeatedFieldsToEntities(true)` in both tests and be done.

Reply:
Right, thanks, good point, reverted the helper in https://github.com/metafacture/metafacture-fix/pull/219/commits/9e0724afc0040cda0986e591cdb5d3f3f8a388d3.